app/editor/src/features/admin/ingests/IngestList.tsx

```diff
@@ -46,22 +132,79 @@ const IngestList: React.FC<IIngestListProps> = (props) => {
         />
       </Row>
       <IngestFilter />
-      <FlexboxTable
-        rowId="id"
-        data={items}
-        columns={columns}
-        showSort={true}
-        onCellClick={async (cell, row) => {
-          if (cell.index !== 6) navigate(`${row.original.id}`);
-          else {
-            await setIngestEnabledStatus(row.original.id, !row.original.isEnabled).catch(() => {});
-            toast.success(
-              `Ingest [${row.original.name}] is now ` +
-                (row.original.isEnabled ? `disabled` : `enabled`),
-            );
-          }
+      <Grid
+        items={items}
+        showPaging={false}
+        onSortChange={(column, direction) => handleOnSorting(column, direction)}
+        renderHeader={() => [
+          { name: 'name', label: 'Name', size: '23%', sortable: true },
+          { name: 'source.code', label: 'Source', size: '10%', sortable: true },
+          { name: 'description', label: 'Description', size: '25%', sortable: true },
+          { name: 'ingestType.name', label: 'Type', size: '10%', sortable: true },
+          { name: 'status', label: 'Status', size: '8%', sortable: true },
+          { name: 'lastRanOn', label: 'Last Run', size: '18%', sortable: true },
+          { name: 'isEnabled', label: 'Enabled', sortable: true },
+        ]}
+        renderColumns={(row: IIngestModel, rowIndex) => {
+          return [
+            {
+              column: (
+                <div key="" className="clickable" onClick={() => navigate(`${row.id}`)}>
```

Comment:
For some reason React will get mad with the `key=""`.  You can just put a unique number in each one to get rid of the console error.

Reply:
ok I will update. thanks